library/src/com/google/maps/android/geoJsonLayer/GeoJsonLayer.java

```diff
@@ -0,0 +1,293 @@
+package com.google.maps.android.geoJsonLayer;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+/**
+ * A class that allows the developer import GeoJSON data, style it and interact with the imported
+ * data.
+ */
+public class GeoJsonLayer {
+
+    private final HashMap<GeoJsonFeature, Object> mFeatures;
```

Comment:
I think this should just be a List (or Set) of GeoJsonFeatures.  I don't think this class should know about the rendered objects.  That Map should be internal to the Renderer.